src/mesh/index_derivs.cxx

```diff
@@ -475,8 +475,9 @@ class FFTDerivativeType {
     AUTO_TRACE();
     throw BoutException("The FFT METHOD isn't available in upwind/Flux");
   }
-  metaData meta{"FFT", 0, DERIV::Standard};
+  static constexpr metaData meta{"FFT", 0, DERIV::Standard};
 };
+constexpr metaData FFTDerivativeType::meta;
```

Comment:
What is it that needs these extra definitions?

Reply:
This is a good explanation: https://stackoverflow.com/a/46719572/2043465
Essentially it's a defect in C++11 that was fixed in C++17.

Also, I doubled checked: the workaround does require both `func` and `meta` to be `constexpr` (and hence `static`).